### PR TITLE
Bintray deployment

### PR DESCRIPTION
**Pull request by weakcamel**
Automate deployment of packages to bintray
Secure credentials in env (.travis.yml)
Deployment script for deb and rpm as well as a wrapper
Remove the deployment provider configs and deploy to PyPI (test/regular) using curl

### CODE REVIEW

**Comment by weakcamel** [comment]
Test releases published to: https://test.pypi.org/project/gcg/#history

See also: https://github.com/nokia/git-changelog-generator/issues/9 (package RPM/DEB **not** using setuptools).

@neechbear if you feel like it, your comments would be most welcome

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,62 @@
-language: python
-install: pip install -r requirements_setup.txt -r requirements_test.txt
+---
+language: bash
+sudo: required
+dist: xenial
+
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y "rpm" "debhelper" python-all fakeroot python-setuptools
```
Comment (weakcamel) [inline .travis.yml]: fix versions and skip the quotes if need be

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+  jq
```
Comment (philiprbrown) [inline .travis.yml]: This looks odd on a line on its own. Do you need to make a YAML multi-line string with '>'?

**Review by philiprbrown — APPROVED** [review]
Just one question above about jq. Then ship-it!

File: .travis.yml (hunk 1 above)
Comment (weakcamel) [reply .travis.yml]: Resolved

File: .travis.yml (hunk 2 above)
Comment (weakcamel) [reply .travis.yml]: Yes, I do. Done, testing.

**Comment by weakcamel** [comment]
Added a tiny script to trigger Bintray repositories metadata recalculation as it appeared it got stale after a test deployment. Tested that manually.

Merging!

**Pull request merged**